ATSQuadroStrategyBase/NinjaTrader 8/bin/Custom/Strategies/AlgoSystemBase.cs

```diff
@@ -3288,14 +3288,14 @@ public virtual StrategyTradeWorkFlowState ProcessWorkFlow(StrategyTradeWorkFlowS
                     }
                     return ProcessWorkFlow(StrategyTradeWorkFlowState.CycleComplete);
                 case StrategyTradeWorkFlowState.ExitOnCloseOrderPending:
-                    if (IsHistoricalTradeOrPlayBack) break;
+                    if (IsHistorical) break;
 
                     TradeWorkFlow = StrategyTradeWorkFlowState.ExitOnCloseWaitingConfirmation;
                     TradeWorkFlowOnMarketDataEnable();
 
                     break;
                 case StrategyTradeWorkFlowState.ExitOnCloseWaitingConfirmation:
-                    if (IsHistoricalTradeOrPlayBack) break;

```

Comment:
This is required - IsHistoricalTradeOrPlayBack use the same route here as backtest - no realtimme workflow required